src/index.js

```diff
@@ -1,12 +1,59 @@
 import "./style.css";
+import { createElement } from "./utils/elements";
+
+var startInterval;
```

Comment:
```suggestion
let startInterval= null;
```
`null` indicates that you will set a value soon.
https://javascript.info/types#the-null-value

Reply:
MDN has an example using var… is it just old or do they use it with purpose?
https://developer.mozilla.org/de/docs/Web/API/WindowOrWorkerGlobalScope/setInterval#Examples